src/EnergyPlus/ReportCoilSelection.cc

```diff
@@ -1478,14 +1502,45 @@ void ReportCoilSelection::setCoilHeatingCapacity(
         }
         // coil entering conditions depend on the type of zone equipment involved
         // set typeof_Coil integer
-        if (DataSizing::TermUnitIU) { // an unpowered induction terminal unit
-            // should be picked up by HeatingWaterDesAirInletTempSizing and HeatingWaterDesAirInletHumRatSizing
-
```

Comment:
This next section is new to update coil inlet temps if they were not set elsewhere. The electric or gas heating coil in a terminal unit is an example.